### ISSUE TITLE
Export models to Onnx

### ISSUE THREAD

**Issue by zcase** [utiasDSL/safe-control-gym#162]
Been trying to export the ppo models to an onnx model but keep running into errors with how it might be implemented. Have any of you exported the trained models to onnx format?

**Comment by Federico-PizarroBejarano** [comment]
Hi @zcase, no I am afraid I haven't, and to my knowledge no one else has. I am not familiar with onnx at all unfortunately. What is the benefit of onnx, and why are you using it?

**Comment by zcase** [comment]
@Federico-PizarroBejarano Onnx is a model format that allows you to transfer in between different frameworks like TensorFlow and Torch. I have found it really helpful when transitioning things to c++. Torch has a way to export to onnx models really easily, however when I tried it with the PPO controller models. It looked like there were some errors. I did modify somethings and was able to get an export but it may have broken the rest of the algorithm when doing training.
